hw/nfc-nci.c

```diff
@@ -713,22 +691,40 @@ init_process_oid_rf_deactivate_cmd(const union nci_packet* cmd,
             rfst = NUMBER_OF_NFC_RFSTS;
     }
 
-    // According to NCI Spec Figure 10.
+    rfst = nfc_rf_state_transition(&nfc->rf_state, bits, rfst);
+    assert(rfst != NUMBER_OF_NFC_RFSTS);
+}
+
+static ssize_t
+nfc_delivery_deactivate_cmd_cb(void* data, union nci_packet* pkt)
+{
+    struct nfc_deactivate_ntf_param* param;
+    struct nfc_device* nfc;
+    ssize_t res = 0;
```

Comment:
Nit: please move this line to the end of the declaration block. Sometimes C compilers are picky about mixing declaration and initialization of variables.